packages/react-table/src/docs/examples/Data.jsx

```diff
@@ -0,0 +1,917 @@
+/* eslint-disable no-console */
+import * as React from 'react';
+import _ from 'lodash';
```

Comment:
We've been trying to avoid using lodash as much as we can in the react-core code. It's a dependency we want to eventually remove.

Reply:
Thank you for the heads-up! Removed in favor of `Math.random`.